src/core/column-type/file.column.d.ts

```diff
@@ -0,0 +1,20 @@
+import {ColumnView} from './column.model.view';
+import {DataColumnModel} from './data.column.model';
+import {IEditorOptions} from './email.column';
+
+export declare interface IHasPreview {
```

Comment:
In this case it's only return function interface, it'll be overkill to have separate files for that